src/saha_boltzmann.jl

```diff
@@ -38,7 +38,7 @@ function saha(χs, Us, T, nₑ)
         k_eV = kboltz_eV
         h = hplanck_cgs
         weights[i] = (weights[i-1]/nₑ * (Us[i](T)/Us[i-1](T)) * (2π*mₑ*k*T/h^2)^1.5 * 
-                      exp(-χs[i]/k_eV/T))
+                      exp(-χs[i-1]/k_eV/T))
     end
     weights ./ sum(weights)
 end
```

Comment:
In some future pull request, I will rewrite this to return a tuple rather than a vector.  Thanks for spotting this bug.